closure/goog/useragent/useragent.js

```diff
@@ -207,8 +194,12 @@ goog.userAgent.SAFARI = goog.userAgent.WEBKIT;
  * @private
  */
 goog.userAgent.determinePlatform_ = function() {
-  var navigator = goog.userAgent.getNavigator();
-  return navigator && navigator.platform || '';
+  const nav = goog.userAgent.getNavigator();
```

Comment:
ES6 is not used in this library at the moment since this would make it impossible to test against older browsers without compiling and compiling the test cases seems to be an issue.